signer/src/transaction_signer.rs

```diff
@@ -425,6 +430,14 @@ where
     ) -> Result<(), Error> {
         let db = self.context.get_storage_mut();
 
+        if self.used_presign_blocks.contains(chain_tip) {
+            tracing::debug!(
+                ?chain_tip,
```

Comment:
The bitcoin chain tip block hash is already in our logs, no need to add it again.